coax/proba_dists/_normal.py

```diff
@@ -101,10 +127,16 @@ def log_proba(dist_params, X):
             mu = check_shape(dist_params['mu'], name='mu', flatten=True)
             logvar = check_shape(dist_params['logvar'], name='logvar', flatten=True)
 
+            if self.squash:
+                X = jnp.arctanh(X)
```

Comment:
Is this numerically stable? Should we perhaps clip `X` before feeding it into an arctanh?

Reply:
Yes, you are right. I will clip it to the range `[-1.0 + 1e-7, 1.0 - 1e-7]` as this is the range where the expression does not evaluate to `inf`.

Reply:
Thanks!